libnmstate/nm/bridge.py

```diff
@@ -139,11 +153,17 @@ def _get_bridge_setting(nmdev):
     return bridge_setting
 
 
-def _get_bridge_ports_info(port_profiles):
+def _get_bridge_ports_info(port_profiles, show_vlan_info=False):
     ports_info = []
     for p in port_profiles:
         port_info = _get_bridge_port_info(p)
         if port_info:
+            if show_vlan_info:
+                port_info[
+                    LB.Port.VLAN_SUBTREE
+                ] = PortVlanFilter.build_from_port_vlans(
+                    p.get_setting_bridge_port().props.vlans
+                ).to_dict()
```

Comment:
Code format here is quite ugly it would be nice if you could manage to fix it. Please consider this as a nit, not a must :-)

Reply:
+ 1
It is hard to read

The usage of `PortVlanFilter.build_from_port_vlans` is also strange. The object represent a port but here we have a static method that gets the vlans from the setting. I would expect to instantiate the object and let it import its state from an existing setting.

Reply:
I can store `p.get_setting_bridge_port().props.vlans` in a variable. Hopefully that'll make it look better.

> The usage of PortVlanFilter.build_from_port_vlans is also strange. The object represent a port but here we have a static method that gets the vlans from the setting. I would expect to instantiate the object and let it import its state from an existing setting.

The advantage there being ? I just don't see it.

You prefer:
```python
a = PortVlanFilter(None, None, None)
a.import(p.get_setting_bridge_port().props.vlans)
port_info[...] = a.to_dict()
```
to 
```python
port_info[...] = PortVlanFilter.build_from_port_vlans(
    p.get_setting_bridge_port().props.vlans
) 
```
? The relevant point (to me) is the intermediate object is useless: It only serves the purpose of translating from NM data to dict and the other way around.

Would renaming build_from_port_vlans to 'import_from_NM' be a compromise between both ?